packages/ember-glimmer/lib/syntax/let.ts

```diff
@@ -0,0 +1,45 @@
+import { compileList } from "@glimmer/runtime";
+
+/**
+@module ember
+
+/**
+  The `let` helper receives one or more positional arguments and yields
+  them out as block params.
+
+  This allows the developer to introduce shorter names for certain computations
+  in the template.
+
+  This is especially useful if you are passing properties to a component
+  that receives a lot of options and you want to clean up the invocation.
+
+  For the following example, the template receives a `post` object with
+  `content` and `title` properties, and a `user` object with `firstName`
```

Comment:
I don't see a `user` object passed in the example below, maybe that was from a prior iteration?